db/structure.sql

```diff
@@ -120,4 +120,74 @@ CREATE TABLE `settings` (
   PRIMARY KEY (`id`),
   INDEX `idx_key` (`key`)
 ) ENGINE=InnoDB AUTO_INCREMENT=101 DEFAULT CHARSET=utf8mb4;
-/*!40101 SET character_set_client = @saved_cs_client */;
\ No newline at end of file
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `capabilities`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `capabilities` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `is_system_supplied` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `groups`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `groups` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `is_system_supplied` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `people_groups`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `people_groups` (
+  `people_id` varchar(255) NOT NULL,
+  `group_id` varchar(255) NOT NULL,
+  KEY `idx_people_id` (`people_id`),
+  KEY `idx_group_id` (`group_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `group_capabilities`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `group_capabilities` (
+  `group_id` varchar(255) NOT NULL,
+  `capabilities_id` varchar(255) NOT NULL,
+  `params1` varchar(255) DEFAULT NULL,
+  `params2` varchar(255) DEFAULT NULL,
+  `params3` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
```

Comment:
We do not have an `id` column, so we have a multi-column primary key
```suggestion
  PRIMARY KEY (`group_id`, `capabilities_id`),
```